content/docs/tasks/security/plugin-ca-cert/index.md

```diff
@@ -42,6 +42,14 @@ Note that if your `ca-cert.pem` is the same as `root-cert.pem`, the `cert-chain.
 
 These files are ready to use in the `samples/certs/` directory.
 
+  {{< tip >}}
+  The default Citadel installation mounts the `cacert` secret as a volume under `/etc/certs`.
+  It is configured via [command line options](/docs/reference/commands/istio_ca/index.html)
```

Comment:
This sentence is a little confusing because if you click on the link and look at the command line options, the default is ``. Maybe rewording slightly would be better?

The default installation sets [command line options](/docs/reference/commands/istio_ca/index.html)
to configure the location of certificates and keys to the predefined file names used ...

Reply:
rephrased to clarify. PTAL and lgtm if acceptable